server/sv_boss.lua

```diff
@@ -138,6 +140,34 @@ RegisterNetEvent('qb-bossmenu:server:FireEmployee', function(target)
 	TriggerClientEvent('qb-bossmenu:client:OpenMenu', src)
 end)
 
+-- Toggle Duty
+RegisterNetEvent('qb-bossmenu:server:toggleEmployeeDuty', function(target)
+	local src = source
+	local Player = QBCore.Functions.GetPlayer(src)
+	local Target = QBCore.Functions.GetPlayerByCitizenId(target)
+
+	if not Player.PlayerData.job.isboss then
```

Comment:
Add a job check to this to help secure the event, ensuring that those modifying the employee's duty have the same job.